DESCRIPTION

```diff
@@ -27,10 +27,14 @@ BugReports: https://github.com/inbo/camtrapdp/issues
 Imports:
     cli,
     dplyr,
+    EML,
     frictionless (>= 1.1.0),
     memoise,
     purrr,
-    readr
+    readr,
+    rlang,
+    tidyr,
```

Comment:
@sannegovaert This is not a light dependency and I notice we only use the `separate()` function. Is it possible to refactor this, so we don't need tidyr?

Reply:
Done